xrpl/asyncio/transaction/reliable_submission.py

```diff
@@ -113,3 +114,32 @@ async def send_reliable_submission(
     return await _wait_for_final_transaction_outcome(
         transaction_hash, client, prelim_result, 0
     )
+
+
+async def submit_and_wait(
```

Comment:
The xrpl.js version of this transaction has support for signing and submitting or just signing - I'd rather this method do the same thing.

Reply:
https://github.com/XRPLF/xrpl.js/blob/main/packages/xrpl/src/sugar/submit.ts#L50-L93
The submitAndWait in xrpl.js will submit the transaction regardless, or am I missing something here? Probably just add the `opt` variable and handling?

Reply:
Sorry, I misspoke. `submitAndWait` in xrpl.js has two modes - just submit an already signed transaction, or autofill/sign/submit. This method should support both.